tests/stress/CMakeLists.txt

```diff
@@ -8,24 +8,39 @@ if (BUILD_ENCLAVES)
 endif ()
 
 # stress test case definition:
-# add_enclave_test(tests/your_stress_test_name stress_host stress_enc stress_test_type enclave_count operation_count)
+# For create enclave & do ecall:
+#   add_enclave_test(tests/your_stress_test_name stress_host stress_enc stress_test_type enclave_count do_ecall_count)
+# For memory management:
+#   add_enclave_test(tests/your_stress_test_name stress_host stress_enc stress_test_type do_memory_count memory_size)
+# For multi-thread:
+#   add_enclave_test(tests/your_stress_test_name stress_host stress_enc stress_test_type thread_count enclave_count)
```

Comment:
Does threadcount always have to match enclave count?

Reply:
I think each thread creates an encalve_count number of enclaves. Closing this.

---
In reply to: [495161827](https://github.com/openenclave/openenclave/pull/3558#discussion_r495161827) [](ancestors = 495161827)